src/lib/sveltekit/hooks/posthog.ts

```diff
@@ -0,0 +1,40 @@
+import { error, json, text, type Handle } from '@sveltejs/kit'
+
+const POSTHOG_URL = 'https://eu.i.posthog.com'
+const PROXY_ROUTE = '/api/track/ingest'
```

Comment:
```suggestion
const PROXY_ROUTE = '/api/track/posthog'
```
To clearly identify the role of this subpath

Reply:
Anyway, it's like we can test this right away, and maintaining a uniform routing style is a good idea

Reply:
fixed